FFMpegCore/FFMpeg/Arguments/DemuxConcatArgument.cs

```diff
@@ -16,8 +16,17 @@ public class DemuxConcatArgument : IInputArgument
         public readonly IEnumerable<string> Values;
         public DemuxConcatArgument(IEnumerable<string> values)
         {
-            Values = values.Select(value => $"file '{value}'");
+            Values = values.Select(value => $"file '{Escape(value)}'");
         }
+
+        /// <summary>
+        /// Thanks slhck
+        /// https://superuser.com/a/787651/1089628
+        /// </summary>
+        /// <param name="value"></param>
+        /// <returns></returns>
+        private string Escape(string value) => value.Replace("'", @"'\''");
```

Comment:
Looks like you are inserting two unespaced single quotes when replacing here

Reply:
Works on my machine where the source has a ``'`` in its path.
Sln form https://superuser.com/a/787651/1089628

Reply:
@rosenbjerg 
``file 'Heaven's River\05 - Investigation.m4b'``
Becomes
``file 'Heaven'\''s River\05 - Investigation.m4b'``